src/pages/en/guides/cms/storyblok.mdx

```diff
@@ -192,18 +192,16 @@ export default defineConfig({
 
 ### Fetching data
 
-To test the setup, in Storyblok create a new story with the `blogPost` content type named `test-post`. 
+To test the setup, create a new page in the `src/pages` directory named `index.astro` with the following content:
 
-In Astro, create a new page in the `src/pages` directory named `test-post.astro` with the following content:
-
-```astro title="pages/test-post.astro"
+```astro title="pages/index.astro"
 ---
 import { useStoryblokApi } from '@storyblok/astro'
 import StoryblokComponent from '@storyblok/astro/StoryblokComponent.astro'
 
 const storyblokApi = useStoryblokApi()
 
-const { data } = await storyblokApi.get("cdn/stories/test-post", {
+const { data } = await storyblokApi.get("cdn/stories/home", {
   version: import.meta.env.DEV ? "draft" : "published",
 });
 
```

Comment:
Maybe we need a disclaimer at the end stating that this fetch will only work if you have a story named `home`

Reply:
We do say to create `home` earlier

Reply:
My bad - I meant to only make this change in the **Making a blog** section. Reverting this one